### DISCUSSION TITLE
no good images in view.

### DISCUSSION THREAD

**Discussion by sfjohnston** [cdcseacave/openMVS#815] [Q&A]
No matter what I try, I keep getting the following message:

[TIME] [App     ] error: reference image  [NUMBER] has no good images in view

I have generated the .mvs files from OpenMVG.
When I open the .mvs file in Viewer, I can see data, and can cycle through cameras and 'shift-V' shows that some points are seen from the views.
The data is sparse. (I have one good high-res image, and a low-res video as source.)
The video frames don't have intrinsics, so on the OpenMVG side, I've had to use the "-f" option to guess a focal-length in pixels.
Any hints as to how I can get Densify to work?

**Comment by cdcseacave** [comment]
if some views do not have good images in view that means that those images do not have any good neighbor images appropiate for densification
however the densification should work with the rest of the images
try to get better images next time: check the internet on how to get good images for photogrsmmetry

**Comment by sfjohnston** [comment]
Thanks!
Unfortunately, the video is of a location that can no longer be captured.
I will explore other "pairing" options to see if I can get better results.
(Also inserting color harmonization into the workflow.)
